### ISSUE TITLE
Unexpected behaviour when re-generating robocop_todo for Metric cops

### ISSUE THREAD

**Issue by Ana06** [bbatsov/rubocop#4156]
When having a `.rubocop.yml` file with the following code:

``` Ruby
Metrics/LineLength:
  Max: 150
```
--------

## Actual behavior

if we add new files to the project where one of then has a line with more than 150 characters (let's say 200) and we want to re-generate the `rubocop_todo.yml` file, that's what we get:

``` Ruby
Metrics/LineLength:
  Max: 200
```

## Expected behavior

I expected to get:

``` Ruby
Metrics/LineLength:
  Max: 150
  Exclude:
     - 'file'
```

If not from that moment all in the rest of the files Rubocop won't complain of lines between 150 and 200 characters, what it did before. :confounded: 

## RuboCop version

```
0.47.1 (using Parser 2.4.0.0, running on ruby 2.2.5 x86_64-linux-gnu)
```

**Comment by Ana06** [comment]
I guess this is wanted, so I'll send a PR. It will be really useful for projects with offences that give a huge max. 😉

**Comment by elliotb** [comment]
This would be really helpful when applying rubocop to legacy projects

**Comment by elliotb** [comment]
This is also related to #3264

**Comment by garettarrowood** [comment]
^ That is more related.  This is slightly related, in that it outlines some "breaking" todo generation behaviour with `Metrics/LineLength:Max` since November.

**Comment by Ana06** [comment]
I would say that #5422 is not related. Solving #3264 may help fixing #3264 though. 

I think I can add an option to fix #3264 when fixing this.

I will send a PR then! 😉

**Comment by deivid-rodriguez** [comment]
[This old feature proposal](https://github.com/bbatsov/rubocop/issues/2424) could also be related, just linking to it in case @Ana06 likes it and wants to take a stab at it :stuck_out_tongue_closed_eyes:.

**Comment by e1senh0rn** [comment]
Encountered same issue with `Layout/SpaceBeforeBlockBraces` cop. Got the following autogenerated in rubocop_todo:
```yaml
Layout/SpaceBeforeBlockBraces:
  EnforcedStyleForEmptyBraces: no_space
```

**Comment by tibbon** [comment]
@e1senh0rn I've had this happen too. Instead of disabling the `Layout/SpaceBeforeBlockBraces` it gives it as `no_space` which doesn't work as I have mixed-styles in the project right now.

**Comment by e1senh0rn** [comment]
Currently I am using this workaround. It generates output completely ignoring counters.
```ruby
# lib/rubocop_simple_format.rb
module RuboCop
  module Formatter
    class DisabledConfigFormatter
      def output_cop_config(output, _cfg, cop_name)
        output.puts "#{cop_name}:"
        output_offending_files(output, {}, cop_name)
      end
    end
  end
end
```

and then in rubocop.yml:
```yaml
require:
  - ./lib/rubocop_simple_format
...
```
